linkerd/app/outbound/src/http/logical/policy/route.rs

```diff
@@ -109,6 +110,8 @@ where
                 // consideration, so we must eagerly fail requests to prevent
                 // leaking tasks onto the runtime.
                 .push_on_service(svc::LoadShed::layer())
+                // Sets an optional request timeout.
+                .push(http::NewTimeout::layer())
```

Comment:
I thought we supported per-backend filters. Am I wrong? Or is that missing? Or is it out of scope?

Reply:
You mean per-backend timeouts? Those are implemented in a separate PR, #2419, as I discussed in the PR description.